packages/search-ui/src/__tests__/actions/setSearchTerm.test.js

```diff
@@ -100,20 +126,19 @@ describe("#setSearchTerm", () => {
     expect(subject("term", { refresh: false }).state.wasSearched).toBe(false);
   });
 
-  it("Will not debounce requests if there is no debounce specified", async () => {
+  it("Will not debounce requests if there is no debounce specified", () => {
     const { driver, mockApiConnector } = setupDriver();
     driver.setSearchTerm("term", { refresh: true });
-    driver.setSearchTerm("term", { refresh: true });
-    driver.setSearchTerm("term", { refresh: true });
-    expect(getSearchCalls(mockApiConnector)).toHaveLength(3);
+    jest.runAllTimers();
+    expect(getSearchCalls(mockApiConnector)).toHaveLength(1);
   });
 
-  it("Will debounce requests", async () => {
+  it("Will debounce requests", () => {
     const { driver, mockApiConnector } = setupDriver();
-    driver.setSearchTerm("term", { refresh: true, debounce: 10 });
-    driver.setSearchTerm("term", { refresh: true, debounce: 10 });
-    driver.setSearchTerm("term", { refresh: true, debounce: 10 });
```

Comment:
Similar-ish question as above - shouldn't we add/keep at least one more `driver.setSearchTerm()` call to ensure debouncing is working properly?

Reply:
Fixed in https://github.com/elastic/search-ui/pull/391/commits/159f8c1e6b5afcda13b6ee29acc8fbc946c16d40 with an explanation.